frontend/components/welcomeHeader.jsx

```diff
@@ -17,27 +17,58 @@ const TitleContainer = styled.div`
 `;
 
 const TimerContainer = styled.div`
-  align-self: flex-end;
+`;
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled.div`
+`;
+
+const SolvedContainer = styled.div`
 `;
 
 class WelcomeHeader extends React.Component {
-  render() {
-    const { gameState } = this.props;
+  constructor(props) {
+    super(props);
+    this.renderSolved = this.renderSolved.bind(this);
+  }
 
-    const { start_time } = gameState || {};
+  renderSolved(gameState) {
+    var solvedCounter = 0;
+    const gameStateValues = Object.values(gameState);
```

Comment:
this can be simplified a little bit with reduce! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce
```
return Object.values(gameState).reduce((solved, info) => {
  return info['solved'] ? solved + 1 : solved;
}, 0);
```